client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -0,0 +1,141 @@
+import { NoticeOptions } from 'calypso/state/notices/types';
+import { getProductBySlug, getProductsList } from 'calypso/state/products-list/selectors';
+import canUserPurchaseGSuite from 'calypso/state/selectors/can-user-purchase-gsuite';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import { getDomainsWithForwards } from 'calypso/state/selectors/get-email-forwards';
+import { fetchSiteDomains } from 'calypso/state/sites/domains/actions';
+import { getDomainsBySiteId, isRequestingSiteDomains } from 'calypso/state/sites/domains/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import EmailProvidersStackedCard from './email-provider-stacked-card';
+import { professionalEmailCard } from './provider-cards/professional-email-card';
+
+import './style.scss';
+
+type Site = {
+	ID: number;
+	slug: string;
+};
+
+type EmailProvidersStackedComparisonProps = {
+	comparisonContext: string;
+	selectedDomainName: string;
+	source: string;
+	cartDomainName?: string;
+	selectedSite?: Site;
+};
+
+export interface ProviderCard {
+	additionalPriceInformation?: TranslateResult;
+	badge?: ReactElement;
+	buttonLabel?: TranslateResult;
+	children?: ReactElement;
+	description: TranslateResult;
+	detailsExpanded: boolean;
+	discount?: string;
+	expandButtonLabel: TranslateResult;
+	features: TranslateResult[];
+	footerBadge?: ReactElement;
+	formattedPrice: string;
+	formFields: ReactElement;
+	logo: ReactElement;
+	onExpandedChange: ( providerKey: string, expanded: boolean ) => void;
+	onButtonClick?: ( event: React.MouseEvent ) => void;
+	productName: TranslateResult;
+	providerKey: string;
+	showExpandButton: boolean;
+}
+
+const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedComparisonProps > = (
+	props
+) => {
+	const translate = useTranslate();
+	const professionalEmail: ProviderCard = professionalEmailCard;
+	const { selectedSite } = props;
+
+	const formFields = <p>Placeholder</p>;
+
+	return (
+		<>
+			<QueryProductsList />
+
+			{ selectedSite && <QuerySiteDomains siteId={ selectedSite.ID } /> }
+
+			<h1 className="email-providers-stacked-comparison__header wp-brand-font">
+				{ translate( 'Pick and email solution' ) }
+			</h1>
+
+			<EmailProvidersStackedCard
+				providerKey={ professionalEmail.providerKey }
+				logo={ professionalEmail.logo }
+				productName={ professionalEmail.productName }
+				description={ professionalEmail.description }
+				detailsExpanded={ professionalEmail.detailsExpanded }
+				discount={ '$42' }
+				additionalPriceInformation={ 'per mailbox' }
+				onExpandedChange={ professionalEmail.onExpandedChange }
+				formattedPrice={ professionalEmail.formattedPrice }
+				formFields={ formFields }
+				showExpandButton={ professionalEmail.showExpandButton }
+				expandButtonLabel={ professionalEmail.expandButtonLabel }
+				features={ professionalEmail.features }
+				footerBadge={ professionalEmail.badge }
+			/>
```

Comment:
I am not sure how much TypeScript will like this, but it would be _much_ cleaner to try something like the following:
```suggestion
			<EmailProvidersStackedCard
				additionalPriceInformation={ translate( 'per mailbox' ) }
				formFields={ formFields }
				{ ...professionalEmail } 
			/>
```